data/schema/ai/aspect_complex.cfg

```diff
@@ -67,10 +67,6 @@
 			value=leader_ignores_keep,passive_leader,passive_leader_shares_keep,simple_targeting,support_villages
 			super="{BASE}~bool"
 		[/case]
-		[case]
```

Comment:
This is wrong, you should only remove `attack_depth` from the list. By removing this, you also render `villages_per_scout` and `recruitment_randomness` invalid according to the schema.

Reply:
Whoops!  Fixed in f53e3b9